pymeasure/instruments/oxfordinstruments/itc503.py

```diff
@@ -26,17 +26,38 @@
 import logging
 from time import sleep, time
 import numpy
+from enum import IntFlag
 
 from pymeasure.instruments import Instrument
 from pymeasure.instruments.validators import strict_discrete_set, \
     truncated_range, strict_range
 
+from .adapters import OxfordInstrumentsAdapter
+
 
 # Setup logging
 log = logging.getLogger(__name__)
 log.addHandler(logging.NullHandler())
 
 
+def pointer_validator(value, values):
+    """ Provides a validator function that ensures the passed value is
+    a tuple or a list with a length of 2 and passes every item through
+    the strict_range validator.
+
+    :param value: A value to test
+    :param values: A range of values (passed to strict_range)
+    :raises: TypeError if the value is not a tuple or a list
+    :raises: IndexError if the value is not of length 2
+    """
+
+    if not isinstance(value, (list, tuple)):
+        raise TypeError('{:g} is not a list or tuple'.format(value))
+    if not len(value) == 2:
+        raise IndexError('{:g} is not of length 2'.format(value))
+    return tuple(strict_range(v, values) for v in value)
```

Comment:
It could be argued that `strict_range` itself should work like this -- I was a bit confused when I realised that it takes the min and max of the collection, not just first and second value. ^.^

Reply:
The question is, though, if we should change it now, it's surely been in use for a long time.